sarif/src/main/java/com/jetbrains/qodana/sarif/baseline/Baseline.kt

```diff
@@ -0,0 +1,110 @@
+package com.jetbrains.qodana.sarif.baseline
+
+import com.jetbrains.qodana.sarif.baseline.BaselineCalculation.Options
+import com.jetbrains.qodana.sarif.model.Result
+import com.jetbrains.qodana.sarif.model.Result.BaselineState
+import com.jetbrains.qodana.sarif.model.Run
+
+private typealias Fingerprint = String
+private typealias StrictIndex = MultiMap<Fingerprint, Result>
+private typealias LaxIndex = MultiMap<ResultKey, Result>
```

Comment:
probably dont need here result, counter would be enough.

Reply:
Very nice, just found that when using a counter I can drop the `Set` conversion from `undecidedFromBaseline`